rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -0,0 +1,443 @@
+          return Promise.resolve({
+            data: {
+              since: {
+                count: 5,
+              },
+            },
+          });
+        }
+        return Promise.resolve({ data: {} });
+      });
+      const wrapper = await shallowMountActivityList();
+      // Simulate the component's periodic check for new executions
+      wrapper.vm.lastDate = 1; // Set a valid lastDate so loadSince will execute
+      await wrapper.vm.loadSince();
+      await wrapper.vm.$nextTick(); // Wait for all pending updates to finish
+      const sinceCountData = wrapper.find('[data-testid="since-count-data"]');
+      expect(sinceCountData.exists()).toBe(true);
+      expect(sinceCountData.text()).toContain("5 New Results. Click to load.");
+    });
+  });
+
+  describe("trigger bulk actions", () => {
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+    it("trigger bulk actions - delete", async () => {
+      axiosMock.get.mockImplementation((url) => {
+        if (url.includes("eventsAjax")) {
+          console.log("Initial fetch of reports with data:", reports);
+          return Promise.resolve({
+            data: {
+              total: reports.length,
+              offset: 0,
+              auth: { projectAdmin: true, deleteExec: true },
+              bulkEditMode: false,
+              showBulkDelete: true,
+              bulkSelectedIds: ["42"],
+              reports: reports,
+            },
+          });
+        }
+        if (url.includes("running")) {
+          console.log("Initial fetch of running executions");
+          return Promise.resolve({
+            data: {
+              executions: mockRunningExecutions,
+            },
+          });
+        }
+        return Promise.resolve({ data: {} });
+      });
+      const wrapper = await shallowMountActivityList();
+      // Trigger bulk edit mode LineNo: 94 -99
+      const bulkDeleteButton = wrapper.find(
+        '[data-testid="activity-list-bulk-delete"]',
+      );
+      console.log("Bulk delete button exists:", bulkDeleteButton.exists());
+      if (!bulkDeleteButton.exists()) {
+        console.error("Bulk delete button not found in DOM");
+        return;
+      }
+      await bulkDeleteButton.trigger("click");
+      console.log("Triggered bulk delete button click");
+
+      // Select and click checkboxes for deletion (Component line: Line No: 253 - 266 )
+      const checkboxes = wrapper.findAll(
+        '[data-testid="bulk-delete-checkbox"]',
+      );
+      console.log("Number of checkboxes:", checkboxes.length);
+      checkboxes.forEach((checkbox, index) => {
+        const inputElement = checkbox.element as HTMLInputElement;
+        console.log(`Checkbox ${index} value:`, inputElement.value);
+      });
+      await checkboxes.at(0)?.trigger("click");
+      console.log("First checkbox checked");
+      await checkboxes.at(1)?.trigger("click");
+      console.log("Second checkbox checked");
```

Comment:
The checkboxes here are not selected, check: https://test-utils.vuejs.org/api/#setValue